File: ProAgil.Domain/Evento.cs
using System;
using System.Collections.Generic;

namespace ProAgil.Domain {
    public class Evento {
        public int Id { get; set; }
        public string Local { get; set; }
        public DateTime DateEvento { get; set; }
        public string Tema { get; set; }
        public int QtdPessoas { get; set; }
        public string ImageURL { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public List<Lote> Lote { get; set; }
        public List<RedeSocial> RedeSociais { get; set;  }
        public List<PalestranteEvento> PalestranteEventos { get; set; }
    }
}

File: ProAgil-App/src/app/_models/Evento.ts
import { Lote } from './Lote';
import { Palestrante} from './Palestrante';
import { RedeSocial } from './RedeSocial';

export interface Evento {
      id: number;
      local: string;
      dateEvento: Date;
      tema: string;
      qtdPessoas: number;
      imageURL: string;
      telefone: string;
      email: string;
      lote: Lote[];
      redeSociais: RedeSocial[];
      palestranteEventos: Palestrante[];
}
